specs/tilebag_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'minitest/pride'
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/tilebag'
+
+# Get that nice colorized output
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'TileBag' do
+  describe 'initialize' do
+    it 'creates and instance'do
+      tile_bag = Scrabble::TileBag.new
+      tile_bag.must_be_instance_of Scrabble::TileBag
+    end
+
+    it 'contains a collection of tiles' do
+      tile_bag = Scrabble::TileBag.new
+      tile_bag.default_tiles.must_be_instance_of Hash
+    end
+
+    it 'returns 98 tiles ' do
+      tile_bag = Scrabble::TileBag.new
+      tile_bag.default_tiles.values.sum.must_equal 98
+    end
+
+  describe 'draw_tiles(num)' do
+    it 'returns a collection of random tiles' do
+      tile_bag = Scrabble::TileBag.new
+      tile_bag.draw_tiles(4).must_be_instance_of Array
+    end
+
+    it 'removes tiles from the tile bag' do
+      tile_bag = Scrabble::TileBag.new
+      tile_bag.draw_tiles(7)
+      tile_bag.default_tiles.values.sum.must_equal 91
+    end
```

Comment:
I would also like to see tests for `draw_tiles` that answer the following questions:

- Am I able to draw the last tile from the bag?
- What happens if I try to draw more tiles than are currently in the bag?
- What if I try to draw 0 or -3 tiles?